modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/podcast/PodcastCacheServer.kt

```diff
@@ -80,6 +80,10 @@ interface PodcastCacheServer {
     @GET
     suspend fun getShowNotesChapters(@Url url: String): RawChaptersResponse
 
+    @GET
+    @Headers("Cache-Control: max-stale=7776000") // Use cache available for 90 days, if not cached use network
+    suspend fun getTranscript(@Url url: String): Response<ResponseBody>
```

Comment:
OkHttp automatically applies `Etag`, `Cache-Control`, etc logic ([ref](https://futurestud.io/tutorials/retrofit-2-activate-response-caching-etag-last-modified#:~:text=OkHttp%20will%20automatically%20apply%20Etag%2C%20Cache%2DControl%2C%20etc%20logic%20on%20every%20request%20for%20you)), I'm just overriding the max-stale value.